deploy/operator/olm-catalog/pmem-csi-operator/manifests/pmem-csi-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,285 @@
+          "kind": "Deployment",
+          "metadata": {
+            "name": "pmem-csi.intel.com"
+          },
+          "spec": {
+            "deviceMode": "lvm",
+            "nodeSelector": {
+              "storage": "pmem"
+            },
+            "pmemPercentage": 50
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Storage
+    certified: "false"
+    containerImage: intel/pmem-csi-driver:v0.7.0
+    createdAt: 2020-06-25 10:50Z
+    description: An operator for deploying and managing the PMEM-CSI driver
+    repository: https://github.com/intel/pmem-csi/tree/release-0.7
+    support: Intel
+  name: pmem-csi-operator.v0.7.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: Deployment
+      name: deployments.pmem-csi.intel.com
+      resources:
+      - kind: StatefulSet
+        name: ""
+        version: apps/v1
+      - kind: DaemonSet
+        name: ""
+        version: apps/v1
+      - kind: Secret
+        name: ""
+        version: v1
+      - kind: Service
+        name: ""
+        version: v1
+      specDescriptors:
+      - description: 'CSI Driver mode for device management: ''lvl'' or ''direct'''
+        displayName: Device manager to use
+        path: deviceMode
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:advanced
+      version: v1alpha1
+  description: |2
+
+    Intel PMEM-CSI is a [CSI](https://github.com/container-storage-interface/spec)
+    storage driver for container orchestrators like Kubernetes. It makes local
+    persistent memory ([PMEM](https://pmem.io/)) available as a filesystem volume to
+    container applications. It can currently utilize non-volatile memory devices
+    that can be controlled via the [libndctl utility library](https://github.com/pmem/ndctl).
+    *Persistent memory* in this context refers to a non-volatile dual in-line
+    memory module (NVDIMM). See [PMEM-CSI documentation](http://githbu.com/intel/pmem-csi)
+    for more details.
+
+    This is the operator to deploy and manage [PMEM-CSI](https://github.com/intel/pmem-csi)
+    driver on a Kubernetes cluster.
+
+    ## Prerequisite
+    ### Software required
+
+    The recommended minimum Linux kernel version for running the PMEM-CSI driver is 4.15.
+    See [Persistent Memory Programming](https://pmem.io/2018/05/15/using_persistent_memory_devices_with_the_linux_device_mapper.html)
+    for more details about supported kernel versions.
+
+    ### Hardware required
+
+    Persistent memory device(s) are required for operation. However, some development
+    and testing can be done using [QEMU-emulated](https://docs.pmem.io/persistent-memory/getting-started-guide/creating-development-environments/virtualization/qemu)
+    persistent memory devices.
+
+    ### Cluster requirements
+
+    The operator can be installed on any OLM-enabled Kubernetes cluster with version >= 1.11.
```

Comment:
1.11? We only support >= 1.15.

Reply:
Running the operator requires least 1.11, but yes i agree no use of running only the operator where the driver could not be run. Changed to >= 1.15